client/subcomponents/events/EventForm/EventForm.jsx

```diff
@@ -40,89 +40,106 @@ export default function EventForm(props) {
 
   return (
     <>
```

Comment:
because everything is wrapped in `<section>` tags you are probably sweet to remove the empty fragments `<>` around everything